database/fakeData.js

```diff
@@ -0,0 +1,25 @@
+const faker = require('faker');
+
+let data = [];
```

Comment:
You could wrap this in a function in order to not pollute the global scope.

Reply:
Done